packages/manager/apps/pci-instances/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
You can add vite/client in the tsconfig types if you prefer, it would avoid to add a new file